build.gradle

```diff
@@ -224,457 +224,458 @@ project.tasks['check'].finalizedBy integration
 
 
 def sonarExclusions = [
-  '**/com/emc/**',
-  '**/com/service/**',
-  '**/com/synapps/**',
-  '**/com/viqsoultions/**',
-  '**/uk/gov/courtservice/**',
-  '**/uk/gov/addcase/**',
-  'src/main/java/generated/**',
-  '**/uk/gov/hmcts/darts/**/model/**',
-  '**/uk/gov/hmcts/darts/**/config/**',
-  '**/enums/**',
-  '**/DocumentumIdToJwtCache*',
-  '**/CacheValueWithJwt*',
-  '**/com/service/viq/event/**',
-  '**/uk/gov/hmcts/darts/cache/token/service/**',
-  '**/uk/gov/hmcts/darts/test/**'
+        '**/com/emc/**',
```

Comment:
Why the change in indentation across the whole file? I think this change should be removed.

Reply:
The indentation was auto added by the HMCTs code formatter config. 
As such I believe this better aligns to the HMCTs standards :)

Reply:
Ok that's fine. It's 985 changed lines that are not related to the other changes in this PR, which makes it unnecessarily noisy for reviewing.